src/qspdlog_model.hpp

```diff
@@ -46,5 +46,5 @@ class QSpdLogModel : public QAbstractListModel
     std::deque<entry_t> _items;
     std::optional<std::size_t> _maxEntries;
     std::map<std::string, QBrush> _backgroundMappings;
-    std::map<std::string, QBrush> _foregroundMappings;
+    std::map<std::string, QColor> _foregroundMappings;
```

Comment:
Changing to a QColor directly disallows setting a QBrush style, though a style is a very nieche gimmick imo.

Reply:
The thing is, there is no brush for the foreground.

Reply:
Not? I haven't tried it out but [the docs](https://doc.qt.io/qt-6/qt.html#ItemDataRole-enum) say it's a QBrush for Qt::ForegroundRole

Reply:
Nevermind you're right, only the color works, style gets ignored.